### ISSUE TITLE
Active state for Shortcuts

### ISSUE THREAD

**Issue by shalinisubramani** [kenhyuwa/litepie-datepicker#40]
I used custom Shortcuts. There is no active state for Shortcuts.

**Comment by sdoerger** [comment]
Yes, that would be great, if there would be an active class at the element, like in vue-router

**Comment by AkWinGit** [comment]
@shalinisubramani Could you tell me how you implement custom shortcuts
